automation/test.sh

```diff
@@ -363,7 +362,11 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ (cnao|multus) ]]; then
     label_filter='(Multus,Networking,VMIlifecycle,Expose,Macvtap)'
   elif [[ $TARGET =~ sig-network ]]; then
-    label_filter='(sig-network)'
+    if [[ $TARGET =~ sig-network-istio ]]; then
+      label_filter='(sig-network && Istio)'
```

Comment:
So IIUC you want to run ISTIO on the separate lane together with all network tests? I thought we'd run either/or for 1.26.

I wouldn't block on this, just making sure it's what we want.

Reply:
First time using the label selector. The intent is to run one lane only with Istio tests `(sig-network && Istio)`. And another lane with all the other network tests `(sig-network && !Istio)`. Is the query wrong?